integration-tests/maven/src/test/resources-filtered/projects/classic/src/main/java/org/acme/ClasspathResources.java

```diff
@@ -64,24 +63,6 @@ private String assertInvalidExactFileLocation() {
         }
     }
 
-    private String assertMultiRelease() {
-        final String testType = "assert-multi-release-jar";
-        if (System.getProperty("java.version").startsWith("1.")) {
-            return SUCCESS;
-        }
-        try {
-            //this class is only present in multi release jars
-            //for fast-jar we need to make sure it is loaded correctly
-            Class<?> clazz = this.getClass().getClassLoader().loadClass("io.smallrye.context.Jdk9CompletableFutureWrapper");
```

Comment:
We might need another way to test this as I think this test is actually checking that multi-release jars are correctly supported.
Let's discuss this before merging.

/cc @geoand

Reply:
Agree. We need to find some other library that is using MR Jars.

Reply:
Exactly. That was the exact purpose of the test

Reply:
I've changed to an old SR Commons lib without Jakarta, which should serve the MR test purpose.